web/pandas/contribute.md

```diff
@@ -1,12 +1,55 @@
 # Contribute to pandas
 
-_pandas_ is and always will be **free**. To make the development sustainable, we need _pandas_ users, corporate
-or individual, to support the development by providing their time and money.
+_pandas_ is and will always be **free**. To make the development sustainable, we need _pandas_ users, corporate
+and individual, to support the development by providing their time and money.
 
 You can find more information about current developers in the [team page](about/team.html),
 and about current sponsors in the [sponsors page](about/sponsors.html).
-Financial contributions will mainly be used to advance in the [pandas roadmap](about/roadmap.html).
 
-- If your **company or organization** is interested in helping make pandas better, please contact us at [info@numfocus.org](mailto:info@numfocus.org)
-- If you want to contribute to _pandas_ with your **time**, please visit the [contributing page]({{ base_url }}/docs/development/index.html)
-- If you want to support _pandas_ with a **donation**, please use the [donations page](donate.html).
+<section>
+    <div class="container mt-5">
+      <div class="row text-center">
+        <div class="col-md-4">
+          <span class="fa-stack fa-4x">
+            <i class="fas fa-circle fa-stack-2x pink"></i>
+            <i class="fas fa-building fa-stack-1x fa-inverse"></i>
+          </span>
+          <h4 class="service-heading mt-3 font-weight-bold blue">Corporate support</h4>
+          <p class="text-muted">
+            pandas depends on companies using the software to support its development. Hiring
```

Comment:
companies and institutions ? 
It's not only companies that can let employees contribute

Reply:
Yes, that makes sense, changed. Thanks!